libs/martech/attributes.js

```diff
@@ -1,12 +1,10 @@
-export function processTrackingLabels(text, config, charLimit = 20) {
-  if (!config) {
-    import('../utils/utils.js').then((utils) => {
-      // eslint-disable-next-line no-param-reassign
-      config = utils.getConfig();
-    });
+export function processTrackingLabels(text, config, charLimit) {
+  let analyticsValue = text?.replace(/[^\u00C0-\u1FFF\u2C00-\uD7FF\w]+/g, ' ').replace(/^_+|_+$/g, '').trim();
```

Comment:
nit: assign regex to a descriptive const outside func

Reply:
Fixed